src/app/iteration/iteration.component.html

```diff
@@ -11,17 +11,16 @@
         </h3>
       </div>
       <ul class="iteration-expanded-div" [collapse]="isCollapsedCurrentIteration">
-        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'">
+        <li class="active" *ngFor="let iteration of currentIterations" [routerLinkActive]="'selected-iteration'" 
```

Comment:
So, problem with this code is, you can't really right click on that section and copy the URL ot open it in a new tab. So we need an anchor tag for this.